src/components/filteredTodos/filteredTodos.tsx

```diff
@@ -0,0 +1,24 @@
+import { Todo } from '../../types/Todo';
+
+type Props = {
```

Comment:
This type should not be called Props, since this object is not passed as props to component.